src/Altinn.Apps/AppFrontend/react/.eslintrc.json

```diff
@@ -176,6 +176,7 @@
     ],
     "react/jsx-boolean-value": "off",
     "linebreak-style": "off",
-    "import/no-cycle": "off"
+    "import/no-cycle": "off",
+    "no-param-reassign": "off"
```

Comment:
Is this primarily an issue in the Redux toolkit blocks?
If so I do think explicit commenting on those files would make sense, it is generally a very common source of bugs to reassign/mutate params.

Reply:
Yes, primarily an issue with Redux toolkit, although I see that we also have several cases of disabling this rule for various utils methods. I have updated the rule to match the suggestion from Redux toolkit docs, and done some cleanup related to that. Suggest that we create a new issue for looking at the utils methods that explicitly disable this rule, and consider cleaning up this where possible.